tuna/miopen/subcmd/import_configs.py

```diff
@@ -350,25 +350,25 @@ def run_import_configs(args: argparse.Namespace,
   if args.add_benchmark:
     add_benchmark(args, dbt, logger)
     return True
-  if not (args.tag and args.framework and args.fw_version and args.model and
-          args.md_version):
-    logger.error(
-        """Tag, framework & version, model & version arguments is required to \
-                import configurations""")
-    return False
-
-  mid, fid = get_database_id(args.framework, args.fw_version, args.model,
-                             args.md_version, dbt, logger)
-  if mid is None or fid is None:
-    logger.error(
-        'Please use --add_model and --add_framework to add new model and framework'
-    )
-    return False
+  #if not (args.tag and args.framework and args.fw_version and args.model and
```

Comment:
For this code block, will this be a permanent or temporary comment out? It might be a better idea to just remove in total -- to avoid future confusion if its optional.